src/main/java/csv/WriteCSV.java

```diff
@@ -0,0 +1,36 @@
+package main.java.csv;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriteCSV {
+
+    private final static String PEOPLE_DATA = "people.csv";
+
+    public static void main(String[] args) {
+        PersonCSV header = new PersonCSV("Name", "Age", "City");
+        String headerString = header.PersonToString(header);
+
+        PersonCSV person1 = new PersonCSV("Maga", "22", "Kyiv");
+        String personString1 = person1.PersonToString(person1);
+
+        PersonCSV person2 = new PersonCSV("Sasha", "21", "Kyiv");
+        String personString2 = person2.PersonToString(person2);
+
+        PersonCSV person3 = new PersonCSV("Timoshka", "5", "Kyiv");
+        String personString3 = person3.PersonToString(person3);
+
```

Comment:
There is a cool principle SOLID.
Check it out.

Based on the 'S' – 'Single responsibility' principle.
It is better to create the separate method, that will create and return all persons.